src/components/__tests__/articleActions.test.js

```diff
@@ -0,0 +1,17 @@
+import {
+  GET_SPECIFIC_ARTICLE_SUCCESS,
+  CREATE_ARTICLE_SUCCESS
+} from "../../actions/actionTypes";
+import {
+  createArticleSuccess,
+  getSpecificArticle
+} from "../../actions/articlesActions";
+
+describe("action creators", () => {
+  it("should dispatch CREATE_ARTICLE_SUCCESS type", () => {
+    expect(createArticleSuccess({}).type).toEqual(CREATE_ARTICLE_SUCCESS);
```

Comment:
use a mock store to dispatch the actions.

Reply:
On it!